MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -1014,7 +1014,7 @@ - (BOOL)isPlayerInErrorState {
     if (!_player || !_playerItem) {
         return NO;
     }
-    return _player.status == AVPlayerStatusFailed || _playerItem.status == AVPlayerItemStatusFailed;
+    return _player.error || _playerItem.error;
```

Comment:
do we know that these are always fata? Because if they aren't we don't want to track it as an error. That is my one concern with this change

Reply:
it sounds like it should be only populated if it is fatal?
>By default, this value is nil. If a player reaches a [AVPlayer.Status.failed](https://developer.apple.com/documentation/avfoundation/avplayer/status/failed), the system populates this value with an error that describes the failure.

Reply:
That is a really good question. I think I have to research, but we might be able to be a little more selective.